README.md

```diff
@@ -119,3 +119,63 @@ This section applies to Solidity and JavaScript projects alike.
 [solidity style guide]: ./SOLIDITY_STYLE_GUIDE.md
 [soliumrc]: ./.soliumrc.json
 [travisci]: https://travis-ci.org/
+
```

Comment:
All the links should go to the very bottom of the file. Your section should go above the links.

Reply:
@schemar: Corrected! Apologies for not reviewing the existing format before making changes.